general/standards/embedded-coding-standard.rst

```diff
@@ -43,6 +43,11 @@ Formatting
 ===============
 Just use `team-wide clang-format config <https://github.com/waterloo-rocketry/rocketlib/blob/master/.clang-format>`_
 
+Global Variables
+================
+
+All global variables shall be initialized.
```

Comment:
this is already covered in barr standard, do we need to specifically highlight it here too?

Reply:
Which section? I did not found it

Reply:
"all variables must be initialized"
its actually more strict than just globals